res/app/control-panes/dashboard/enrollment/enrollment.css

```diff
@@ -0,0 +1,14 @@
+.stf-shell .shell-results {
+  font-size: 12px;
+  color: #fefefe;
+  background: #444;
+}
+
+.stf-shell .shell-results-empty {
+  font-style: italic;
+}
+
+.stf-shell .shell-input {
```

Comment:
i think you have to rename these, preferably `stf` -> `enrollment` otherwise there will be duplicated definitions in final CSS